oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/CurrentLocationInformation.jsx

```diff
@@ -0,0 +1,125 @@
+// component to show the user where they currently are
+
+import React from "react";
+import { useAxios } from "../hooks/useAxios";
+import { Message, Breadcrumb } from "semantic-ui-react";
+import _ from "lodash";
+import { i18next } from "@translations/oarepo_vocabularies_ui/i18next";
+import { breadcrumbSerialization } from "../../utils";
+import { ErrorComponent } from "./Error";
+import PropTypes from "prop-types";
+import { useFormConfig } from "@js/oarepo_ui/forms";
+
+const NewTopLevelItemMessage = () => (
+  <Message
+    icon="attention"
+    header={i18next.t("newItemMessage")}
+    size="tiny"
+    // don't understand how to reasonably set width for such a component in semantic!!
+    style={{ width: "68%" }}
+  />
+);
+
+const NewChildItemMessage = ({ newChildItemParentId }) => {
+  const { response, loading, error } = useAxios({
+    url: `/api/vocabularies/institutions/${newChildItemParentId}`,
```

Comment:
No hardcoded vocabulary type should be here

Reply:
Will leave this open to be fixed after we enable getting vocabulary type from BE within formConfig.

Reply:
Vocabulary type provided by https://github.com/oarepo/oarepo-vocabularies/pull/68